src/components/nodes/association/association.vue

```diff
@@ -19,7 +19,7 @@
 import { shapes } from 'jointjs';
 import linkConfig from '@/mixins/linkConfig';
 import get from 'lodash/get';
-import associationHead from '@/assets/association-head.svg';
```

Comment:
These are still needed until we decide on whether to expose these via the `updateIconColor` mixin